Documentation/build-master-role.md

```diff
@@ -17,7 +17,8 @@ This is the Google calendar that shows who takes this role when.
 
 ### Nightly Mender QA build
 
-Link:  https://qastatus.mender.io/build-status
+Nigtly pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipeline_schedules
+QA Status:       https://qastatus.mender.io/build-status
```

Comment:
`https://qastatus.mender.io/build-status` should go below in the `Individual pipelines` section. And here we should have `https://qastatus.mender.io/nightlies` instead.

Reply:
Please put `https://qastatus.mender.io/build-status` in the `Individual pipelines` section.